theme/snippets/product-info.liquid

```diff
@@ -33,6 +33,6 @@
     </div>
   </span>
   <div class="none md:flex md:flex-col">
-    {% render 'add-to-cart-container' product: product %}
+    {% render 'add-to-cart-container' isDesktop: true, product: product %}
```

Comment:
Is this always set to true? Shouldn't this be checking the screen width?

Reply:
That's a really good point that made me re-think what I'm doing! I realized I just need to pass a specific selector (`data-mobile-product-add-to-cart-button` or `data-desktop-product-add-to-cart-button`) to differentiate between the two buttons (since they are actually both always there, although hidden at certain screen sizes). 

I realized `isDesktop` wasn't really doing what it sounded like, when I called it `isDesktop`! 

I changed `isDesktop` to `data_selector`, and now both desktop and mobile add to cart buttons pass `data_selector` to the `add-to-cart-container` so we know which one is which. Does that make sense?

Reply:
I misunderstood what was happening as well. I see now that they're both always present so what you pass in as `isDesktop` will always be hard-coded.

I now think `isDesktop` might be a good choice since it leaves all the specific data-selector stuff into the snippet instead of out of it, but I think this is fine too and might be more flexible in case there is another layout in the future.

Reply:
Ok nice, @sepowitz also agrees so will change this back!